docs/manual/build_system/cmake_variables.qbk

```diff
@@ -34,6 +34,7 @@ The options are split into these categories:
 * [link build_system.cmake_variables.HPX_WITH_COMPRESSION_BZIP2 HPX_WITH_COMPRESSION_BZIP2]
 * [link build_system.cmake_variables.HPX_WITH_COMPRESSION_SNAPPY HPX_WITH_COMPRESSION_SNAPPY]
 * [link build_system.cmake_variables.HPX_WITH_COMPRESSION_ZLIB HPX_WITH_COMPRESSION_ZLIB]
+* [link build_system.cmake_variables.HPX_WITH_FORTRAN HPX_WITH_FORTRAN]
```

Comment:
@hkaiser @sithhell How can I prevent this from happening when I commit changes? It seems building the HPX docs will produce stuff like this